doc/source/tutorial/tutorial_string_syntax.ipyml

```diff
@@ -0,0 +1,160 @@
+cells:
+
+- markdown: |
+    # Pythonic VS String Syntax
+
+
+- markdown: |
+    Import the LArray library:
+
+
+- code: |
+    from larray import *
+
+
+- markdown: |
+    Check the version of LArray:
+
+
+- code: |
+    from larray import __version__
+    __version__
+
+
+- markdown: |
+    LArray offers two syntaxes to build axes and make selections and aggregations.
+    The first one is more ``Pythonic`` (uses Python structures) 
+    For example, you can create an *age_category* axis as follows:
+
+
+- code: |
+    age_category = Axis(["0-9","10-17","18-66","67+"], "age_category")
+    age_category
+
+
+- markdown: |
+    The second one consists of using ``strings`` that are parsed.
+    It is shorter to type. The same *age_category* axis could have been generated as follows:
+
+
+- code: |
+    age_category = Axis("age_category=0-9,10-17,18-66,67+")
+    age_category
+
+
+- markdown: |
+    <div class="alert alert-warning">
+      **Warning:** The drawback of the string syntax is that some characters such as `, ; = : .. [ ] >>`
+    have a special meaning and cannot be used with the ``String`` syntax. 
+    If you need to work with labels containing such special characters (when import data from an external source for example), you have to use the ``Pythonic`` syntax which allows to use any character in labels.
+    </div>
+
+
+- markdown: |
+    ## String Syntax
+
+
+- markdown: |
+    ### Axes And Arrays creation
+    
+    The string syntax allows to easily create axes.
+    
+    When creating one axis, the labels are separated using ``,``:
+
+
+- code: |
+    a = Axis('a=a0,a1,a2,a3')
+    a
+
+
+- markdown: |
+    The special syntax ``start..stop`` generates a sequence of labels:
+
+
+- code: |
+    a = Axis('a=a0..a3')
+    a
+
+
+- markdown: |
+    When creating an array, it is possible to define several axes in the same string using ``;``
+
+
+- code: |
+    arr = zeros("a=a0..a2; b=b0,b1; c=c0..c5")
+    arr
+
+
+- markdown: |
+    ### Selection
+    
+    Starting from the array: 
+
+
+- code: |
+    immigration = load_example_data('demography_eurostat').immigration
+    immigration.info
+
+
+- markdown: |
+    an example of a selection using the ``Pythonic`` syntax is:
+
+
+- code: |
+    immigration_subset = immigration[X.country['Belgium', 'Netherlands'], 'Female', 2015:]
```

Comment:
add a comment specifying country needs to be specified because there is another axis with the same labels (I **assume** this is the case)